### DISCUSSION TITLE
Motor / ESC / Prop failure warning

### DISCUSSION THREAD

**Discussion by carlybaby69** [iNavFlight/inav#7976] [Ideas]
A few times when I’ve burnt out an ESC mid flight or temp cut off has happened it can take some vital time to realise this. 

Home about some extra parameters in inav:

If throttle % = <some user defined input>
AND
AMP draw < <some user defined input>

Then put some message on the osd. 

E.g if on my Zohd Alphastrike throttle is 90% and amp draw is 3a then something is clearly wrong. The few seconds in diagnosing issue could mean a successful (manual) return to home

**Comment by MrD-RC** [comment]
You can set an alert on ESC temp. So if it goes over x degrees, it will blink to get your attention.

**Comment by breadoven** [comment]
I take it there was never any way of setting a custom OSD message based on output from the programming ? If there was you could do this using programming. Add it to the System Message perhaps if switched on from programming.
